Dockerfile

```
@@ -1,5 +1,5 @@
 ARG NODE_VERSION=18
-FROM node:${NODE_VERSION}-buster
+FROM node:${NODE_VERSION}
```

might want to pull in an alpine release here so the docker container isn't massive

```suggestion
FROM node:${NODE_VERSION}-alpine3.18
```